spec/features/valkyrie_search_spec.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
-RSpec.describe 'searching', index_adapter: :solr_index do
+RSpec.describe 'searching', index_adapter: :solr_index, valkyrie_adapter: :test_adapter do
```

Comment:
Specifying a `valkyrie_adapter` other than wings ensures that this test works without ActiveFedora.